aws/logs_monitoring/template.yaml

```diff
@@ -161,6 +161,13 @@ Parameters:
       - true
       - false
     Description: Set to false to disable log forwarding, while continuing to forward other observability data, such as metrics and traces from Lambda functions.
+  DdStepFunctionTraceEnabled:
+    Type: String
+    Default: true
```

Comment:
I think the default value should be false, so that customers are not billed unexpectedly.